keps/sig-node/5055-device-taints-and-tolerations/kep.yaml

```
@@ -0,0 +1,41 @@
+title: "DRA: device taints and tolerations"
+kep-number: 5055
+authors:
+  - "@pohly"
```

Would you mind adding me as an author? I’d really appreciate it if you don’t mind, as I believe I contributed to the main part of this design.

```suggestion
authors:
  - "@pohly"
  - "@everpeace"
```

---

> Sure! Of course, 

Thanks!!

> now you are also on the hook to review the implementation and help answer whatever questions people might have about it... 😈

Sure, I'd love to!!